docs/console-screen-buffers.md

```diff
@@ -32,43 +31,42 @@ A number of properties associated with a screen buffer can be set independently
 - Cursor position, appearance, and visibility.
 - Output modes (**ENABLE\_PROCESSED\_OUTPUT** and **ENABLE\_WRAP\_AT\_EOL\_OUTPUT**). For more information about console output modes, see [High-Level Console Modes](high-level-console-modes.md).
 
-When a screen buffer is created, it contains blanks. Its cursor is visible and positioned at the buffer's origin (0,0), and the window is positioned with its upper left corner at the buffer's origin. The size of the console screen buffer, the window size, the text attributes, and the appearance of the cursor are determined by the user or by the system defaults. To retrieve the current values of the various properties associated with the console screen buffer, use the [**GetConsoleScreenBufferInfo**](getconsolescreenbufferinfo.md), [**GetConsoleCursorInfo**](getconsolecursorinfo.md), and [**GetConsoleMode**](getconsolemode.md) functions.
-
-Applications that change any of the console screen buffer properties should either create their own screen buffer or save the state of the inherited screen buffer during startup and restore it at exit.
+When a screen buffer is created, it contains space characters in every position. Its cursor is visible and positioned at the buffer's origin (0,0), and the window is positioned with its upper left corner at the buffer's origin. The size of the console screen buffer, the window size, the text attributes, and the appearance of the cursor are determined by the user or by the system defaults. To retrieve the current values of the various properties associated with the console screen buffer, use the [**GetConsoleScreenBufferInfo**](getconsolescreenbufferinfo.md), [**GetConsoleCursorInfo**](getconsolecursorinfo.md), and [**GetConsoleMode**](getconsolemode.md) functions.
```

Comment:
> [!TIP]
> It is recommended to use the [**alternate buffer mode**](console-virtual-terminal-sequences.md#alternate-screen-buffer) going forward, if possible, instead of creating a second screen buffer for this purpose. **Alternate buffer mode** offers increased compatibility across remote devices and with other platforms. Please see our discussion on [**classic console APIs versus virtual terminal**](classic-vs-vt.md) for more information.